library/src/main/java/com/datatorrent/lib/io/fs/AbstractReconciler.java

```diff
@@ -187,6 +189,18 @@ public void run()
   }
 
   /**
+   * Returns the unmodifiable collection of the tuples on which
+   * processCommittedData has been called in this window. Underlying collection
+   * is cleared at every endWindow() of AbstractReconciler.
+   * 
+   * @return done tuples
+   */
+  protected Collection<QUEUETUPLE> getDoneTuples()
+  {
+    return Collections.unmodifiableCollection(doneTuples);
+  }
```

Comment:
Can you give details about the usecase why you need only read view and not write view? Why even read view is required as I think this is internal to operator?